hollow-diff-ui/src/main/java/com/netflix/hollow/history/ui/webserver/HollowHistoryUIServer.java

```diff
@@ -14,7 +14,7 @@
  *     limitations under the License.
  *
  */
-package com.netflix.hollow.history.ui.jetty;
```

Comment:
There are two HollowHistoryUIServer in jetty and webserver packages to avoid breaking dependencies.
hollow-diff-ui/src/main/java/com/netflix/hollow/history/ui/jetty/HollowHistoryUIServer.java
hollow-diff-ui/src/main/java/com/netflix/hollow/history/ui/webserver/HollowHistoryUIServer.java

Reply:
done